src/system/ThemeModule/Engine/AssetFilter.php

```diff
@@ -14,14 +14,14 @@
 use Zikula\ThemeModule\Engine\Asset\ResolverInterface;
 
 /**
- * Class Filter
+ * Class AssetFilter
  *
  * This class resolves, compiles and renders all page assets and adds them to the outgoing source content
  * Recently, the accepted practice for placement of javascript has changed from header to footer. Placement in this
  * class is determined by the `scriptPosition` parameter in `app/config/parameters.yml`. In Core-1.4 this defaults
  * to the header. In Core-2.0 it will default to footer. Scripts must be written to accommodate this change.
  */
-class Filter
+class AssetFilter
```

Comment:
It contains a method named `filter`. Thus the class should have another name so PHP does not think it is a legacy (PHP4 type) constructor.